browse-scaler/src/index.js

```diff
@@ -67,10 +67,16 @@ const resizeImageFromConceptId = async (conceptType, conceptId, height, width) =
 
   // If given an image url, fetch the image and resize. If no image
   // exists, return the not found response
-  const imageUrl = await getImageUrlFromConcept(conceptId, conceptType);
-  const imageBuffer = await withTimeout(timeoutInterval, slurpImageIntoBuffer(imageUrl));
+  const imageUrl = await withTimeout(timeoutInterval, getImageUrlFromConcept(conceptId, conceptType));
```

Comment:
This is a call to the CMR, which shouldn't ever time out unless we're experiencing degraded service. If that is the case, this may end up caching that an image is not found for a concept that does contain a browse image. I think this line should just be:
```const imageUrl = await getImageUrlFromConcept(conceptId, conceptType);```

Definitely keep the ```if (imageUrl === null) { ... }``` check though. That's good thinking

Reply:
On second thought, it should be okay to leave this - not necessary, as I doubt the CMR will have queries that will ever take this long given a concept-id under normal conditions, but it shouldn't be a _problem_

Reply:
This is added not because of concerning getImageUrlFromConcept will timeout, it's because we don't want any function to take too long. If we don't want to see 502, we have to make sure the request won't time out before all the functions are returned with either the image, or the no image found.